openapi.yaml

```diff
@@ -21985,7 +21994,15 @@ components:
       type: object
       properties:
         meta:
```

Comment:
If we did not annotate this field, the data source would contain an empty `meta` object